src/verifier.ts

```diff
@@ -37,8 +37,7 @@ export class Verifier {
 	constructor(options: VerifierOptions) {
 		options = options || {};
 		options.pactBrokerUrl = options.pactBrokerUrl || "";
-		options.consumerVersionTag = options.consumerVersionTag || "";
-		options.tags = options.tags || [];
+		options.consumerVersionTag = _.toArray(options.consumerVersionTag);
```

Comment:
This will turn a string like `"tag-1"` into an array like `[ 't', 'a', 'g', '-', '1' ]`